### ISSUE TITLE
{Linked:Bug2281690}[Adaptive Card-Designer]: 'Asterisk' is not defined for required field.

### ISSUE THREAD

**Issue by MohitTyagi13** [microsoft/AdaptiveCards#7652]
### Target Platforms

Other

### SDK Version

1.3

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: 22H2 (OS Build 25151.1010)
Browser: Edge Dev
Browser Version: Version 105.0.1300.0 (Official build) dev (64-bit)
URL: [Designer | Adaptive Cards](https://adaptivecards.io/designer/)

Pre-requisites:
Copy the complete .json code in ''Card Payload Editor". (Please refer attachment).

Repro Steps:
Launch the above URL.
Navigate to header control ‘Designer’ using tab key and activate through enter key.
Apply attached SMB.json in Card Payload Editor and Sample Editor.
Check Card in Preview Mode.
Navigate to Card and observe Asterisk is not visible for highlighted label.
Actual Result:
On payments dialog "Recipients and currency" edit field are required but 'Asterisk' is not available with their labels.

Expected result:
 'Asterisk' should be available with "Recipients and currency" labels.

User Impact:
'Asterisk' helps users to understand the importance of edit field. So, if a field is required then asterisk should be defined.

MAS Reference:
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

<img width="582" alt="'Asterisk' is not defined for required field  (1)" src="https://user-images.githubusercontent.com/95915837/177688894-c5f02b62-c228-47b9-8b55-29a738ddaee7.png">
<img width="960" alt="MAS1 3 1 (1)" src="https://user-images.githubusercontent.com/95915837/177688918-26803bb4-5acb-48bb-99eb-8d9cf15b5392.png">

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "body": [
    {
      "color": null,
      "horizontalAlignment": null,
      "isSubtle": false,
      "maxLines": 0,
      "size": null,
      "text": "Payment provider",
      "weight": null,
      "wrap": true,
      "id": "paymentProviderLabel",
      "separator": false,
      "type": "TextBlock"    },
    {
      "color": null,
      "horizontalAlignment": null,
      "isSubtle": true,
      "maxLines": 0,
      "size": null,
      "text": "Stripe (PoC)",
      "weight": "bolder",
      "wrap": true,
      "id": "paymentProviderName",
      "separator": false,
      "type": "TextBlock"    },
    {
      "color": null,
      "horizontalAlignment": null,
      "isSubtle": false,
      "maxLines": 0,
      "size": null,
      "text": "Recipients",
      "weight": null,
      "wrap": true,
      "id": "participantsLabel",
      "separator": false,
      "type": "TextBlock"    },
    {
      "choices": [
        {
          "title": "Test User23",
          "value": "29:1jgdJQy14x2o_zTdruaptPsqhFw3lTIMhaoJT7RhOri46DKTpLw9clrWyBxcKgaHZEftLgVqJEGc5xcZUBs9AFA"        }
      ],
      "isMultiSelect": true,
      "style": null,
      "placeholder": "Select one or more recipients",
      "isRequired": true,
      "errorMessage": "Select at least one recipient",
      "id": "participants",
      "separator": false,
      "type": "Input.ChoiceSet"    },
    {
      "isMultiline": false,
      "maxLength": 1000,
      "placeholder": "Enter your service name",
      "style": null,
      "label": "Service",
      "id": "service",
      "separator": false,
      "type": "Input.Text"    },
    {
      "columns": [
        {
          "width": "2",
          "items": [
            {
              "min": 1,
              "placeholder": "0.00",
              "isRequired": true,
              "errorMessage": "The price must be greater than zero",
              "label": "Price",
              "id": "price",
              "separator": false,
              "type": "Input.Number"            }
          ],
          "separator": false,
          "type": "Column"        },
        {
          "width": "2",
          "items": [
            {
              "color": null,
              "horizontalAlignment": null,
              "isSubtle": false,
              "maxLines": 0,
              "size": null,
              "text": "Currency",
              "weight": null,
              "wrap": true,
              "id": "currencyLabel",
              "separator": false,
              "type": "TextBlock"            },
            {
              "choices": [
                {
                  "title": "${name}",
                  "value": "CAD"                },
                {
                  "title": "${name}",
                  "value": "USD"                },
                {
                  "title": "${name}",
                  "value": "BRL"                },
                {
                  "title": "${name}",
                  "value": "JPY"                },
                {
                  "title": "${name}",
                  "value": "EUR"                }
              ],
              "isMultiSelect": false,
              "style": null,
              "value": "INR",
              "isRequired": true,
              "errorMessage": "Select a currency",
              "id": "currency",
              "separator": false,
              "type": "Input.ChoiceSet"            }
          ],
          "separator": false,
          "type": "Column"        }
      ],
      "separator": false,
      "type": "ColumnSet"    },
    {
      "actions": [
        {
          "data": {
            "paymentProviderId": "74e1b13c-33cf-48e2-9471-850fdf9bcdb7",
            "action": "submitCharge"          },
          "title": "Send request",
          "type": "Action.Submit"        }
      ],
      "spacing": "extralarge",
      "separator": false,
      "type": "ActionSet"    }
  ],
  "speak": "Configure a payment using Stripe (PoC). Select the recipients you want to charge, describe the charge and the amount, select currency, then press send request.",
  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2"}
```

**Comment by paulcam206** [comment]
This is actually a bug in the card payload itself. As authored, there's no way for Adaptive Cards to know that the `TextBlock` elements preceding the `Input.ChoiceSet` elements are related. Please use `Input.ChoiceSet`'s `label` property to supply a label:

```diff
--- SMB.json    2022-10-04 12:19:47.320132900 -0700
+++ SMB2.json   2022-10-04 12:20:32.793011100 -0700
@@ -28,19 +28,7 @@
       "type": "TextBlock"
     },
     {
-      "color": null,
-      "horizontalAlignment": null,
-      "isSubtle": false,
-      "maxLines": 0,
-      "size": null,
-      "text": "Recipients",
-      "weight": null,
-      "wrap": true,
-      "id": "participantsLabel",
-      "separator": false,
-      "type": "TextBlock"
-    },
-    {
+      "label": "Recipients",
       "choices": [
         {
           "title": "Test User23",
@@ -89,19 +77,7 @@
           "width": "2",
           "items": [
             {
-              "color": null,
-              "horizontalAlignment": null,
-              "isSubtle": false,
-              "maxLines": 0,
-              "size": null,
-              "text": "Currency",
-              "weight": null,
-              "wrap": true,
-              "id": "currencyLabel",
-              "separator": false,
-              "type": "TextBlock"
-            },
-            {
+              "label": "Currency",
               "choices": [
                 {
                   "title": "${name}",
```

This yields the desired behavior:

![image](https://user-images.githubusercontent.com/16614499/193907774-2927bb2f-338c-4139-85df-8a322433b2cb.png)

and after clicking submit button:

![image](https://user-images.githubusercontent.com/16614499/193907867-9ae3d249-64b6-4838-ba87-9ed9205a704d.png)
